plugins/ida/ida_main.ml

```diff
@@ -166,12 +252,70 @@ let checked ida_path is_headless =
        (ida_path/"plugins"/"plugin_loader_bap.py"))  >>| fun () ->
   ida_path
 
+let run_ko_symbol_mapper_pass ida_futures =
+  Project.register_pass ~autorun:true ~name:"ko_symbol_mapper" (fun proj ->
+      match Project.get proj filename with
+      | Some file when String.is_suffix file ".ko" ->
+
+        let id = Data.Cache.digest ~namespace:"ida-brancher"
+            "%s" (Digest.file file) in
+        let lookup = match Brancher_info.Cache.load id with
+          | Some lookup -> lookup
+          | None ->
+            info "Ko_symbol_pass: No caching enabled, using futures!";
+            read_ida_future_list (fst ida_futures.brancher) in
+
+        let simpl_relocs lookup =
+          let (!) = Word.of_int64 ~width:32 in
+          List.fold ~init:[] lookup ~f:(fun acc (addr,_,l) ->
+              match List.hd l with
```

Comment:
`l` contains a list of potential 'else' target jumps. Most of the time, there's only one else branch target. But, IDA can give us more (for instance, for jump tables). There is a separate pass in the works for emitting all the jumps of jump tables, but it is not ready yet. Until then, we roll with the first (and most of the time, only) else branch in the lookup table.